util/find-doc-nits

```diff
@@ -872,6 +872,18 @@ if ( $opt_l || $opt_u || $opt_v ) {
 }
 
 if ( $opt_l ) {
+    # Load in manpages that are known to not exist
+    open FH, "util/missingdocs.txt"
+        or die "Can't open util/missingdocs.txt, $!";
+    while ( <FH> ) {
+        chop;
+        next if /^#/;
+        my $pod = $_;
+        err("$pod in missingdocs.txt but it exists") if -f "doc/$pod";
+        $pod =~ s@man(.)/(.*)\.pod@$2($1)@;
```

Comment:
Why not have in *name*(*n*) for in the file? After all, that's what we're checking, isn't it? That, say, the manual for `whatever(3)` doesn't exist? Why do you want us to go through hoops to rewrite that as `man3/whatever.pod` before putting it in `util/missingdocs.txt`, when we could just put in `whatever(3)` directly?

Reply:
Line 802 and 803 do different things; one checks for the file existing, the substitution turns it into an ```L<>``` reference.  Now, it *is* possible to move all of the entires in missingdocs.txt into manpage-format links and merge it into missingcrypto (which should be named missingapi I guess).  I can do that if you want, but we lose the check in line 802. If I do that, then some mode of find-doc-nits should warn if something is in missing but is documented.

Reply:
https://github.com/openssl/openssl/issues/10681

Reply:
The notion that there is a 1:1 mapping  between manual references and pod files is a *delusion*. It simply doesn't work that way, since every pod file can have multiple names (and thereby be known as multiple manual references). And since the links are checked a few lines down, you do take the lines from this missingdocs.txt into account anyway, so line 882 (not 802) misses the point.

Reply:
Well, these are things that do not exist right now, so it isn't necessarily delusional.  There's several things that have a link to OSSL_STORE.  What API is that?

Reply:
OSSL_STORE is the store API, i.e. crypto/store (dude, where's your grep-fu?).  These are the missing manuals:

```
OSSL_STORE_do_all_loaders(3)
OSSL_STORE_vctrl(3)
```

I would intuitively add `OSSL_STORE_do_all_loaders` to `doc/man3/OSSL_STORE_LOADER.pod` and `OSSL_STORE_vctrl` to `doc/man3/OSSL_STORE_open.pod`

QED

Reply:
What should existing ```L<OSSL_STORE(3)>``` resolve to?

Reply:
It should *probably* be changed to `L<ossl_store(7)>`
I'm not in a position to confirm right now

Reply:
I just checked.  Confirmed.

Reply:
fixed.